src/app.rs

```diff
@@ -95,6 +95,11 @@ pub fn build_app() -> App<'static, 'static> {
                 .takes_value(true)
                 .hidden(true),
         )
+        .arg(
+            arg("exec")
+                .long("exec")
```

Comment:
The short option `-e` is already taken, but we could consider adding `-x` as a short version of `--exec`

Reply:
Alright -- have added "x" as a short option.